src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -636,6 +637,13 @@ def init(self):
             if fcp >= fcp_minimum_threshold and fcp < fcp_maximum_threshold:
                 self.fcp = fcp
 
+    def is_creation_allowed_for_organization(self, organization: Optional[Organization]) -> bool:
+        return features.has(
+            "organizations:performance-issues-render-blocking-assets-detector",
+            organization,
+            actor=None,
+        )
+
```

Comment:
You should also add  is_creation_allowed_for_project as by default is returns False
https://github.com/getsentry/sentry/blob/44c26450369c77597aa7c8cda4b75f7bbd3552ee/src/sentry/utils/performance_issues/performance_detection.py#L528

The following 3 conditions needs to be met (system is handled by the options)
https://github.com/getsentry/sentry/blob/44c26450369c77597aa7c8cda4b75f7bbd3552ee/src/sentry/utils/performance_issues/performance_detection.py#L325

We have been just return True for now
https://github.com/getsentry/sentry/blob/44c26450369c77597aa7c8cda4b75f7bbd3552ee/src/sentry/utils/performance_issues/performance_detection.py#L1662

Reply:
Thanks, added!